### ISSUE TITLE
NGSCHECKMATE_NCM

### ISSUE THREAD

**Issue by LuciaNhuNguyen** [nf-core/sarek#1366]
### Description of feature

I noticed that nf-core/sarek has added a new step, CRAM_SAMPLEQC:BAM_NGSCHECKMATE:NGSCHECKMATE_NCM. I'm curious about how to use it as I couldn't find any documentation or shortcuts. Is this step intended to be used solely in conjunction with bcftools mileup?"

**Comment by maxulysse** [comment]
I think for now, this is the documentation: https://nf-co.re/sarek/3.4.0/docs/output#ngscheckmate

I'm assigning @SPPearce to this issue

**Comment by maxulysse** [comment]
But you're totally right, `NGSCHECKMATE` requires `bcftools mileup` to run beforehand

**Comment by LuciaNhuNguyen** [comment]
Thank you for your quick response. And Happy New Year!

**Comment by SPPearce** [comment]
Yes, some more documentation would be a good idea, Maxime, where should it go?

NGSCheckMate is designed to check that samples come from the same individual (or not), using a set of SNPs. bcftools mpileup is first used to collect the alleles at the SNPs (on a per sample basis), which are then fed into  NGSCheckMate. It is not related to the mpileup variant calling part of sarek.

**Comment by maxulysse** [comment]
I don't see anything for NGSCheckMate in https://nf-co.re/sarek/3.4.0/docs/usage, but I feel it'll be more helpful there than in https://nf-co.re/sarek/3.4.0/docs/output

**Comment by FriederikeHanssen** [comment]
For how to use it, check here: https://nf-co.re/sarek/3.4.0/parameters#tools 

TL;DR: `--tools ngscheckmate`
